resolwe/flow/filters.py

```diff
@@ -65,54 +90,66 @@ def __init__(self, *args, **kwargs):
 class EntityFilter(CollectionFilter):
     """Filter the Entity endpoint."""
 
-    collections = filters.ModelChoiceFilter(queryset=Collection.objects.all())
-    descriptor_completed = filters.BooleanFilter()
+    collection = filters.ModelChoiceFilter(field_name='collections', queryset=Collection.objects.all())
+    descriptor_completed = filters.rest_framework.filters.BooleanFilter(field_name='descriptor_completed')
     tags = TagsFilter()
 
     class Meta(CollectionFilter.Meta):
         """Filter configuration."""
 
         model = Entity
-        fields = BaseResolweFilter.Meta.fields + ['collections', 'descriptor_completed', 'tags']
+        fields = CollectionFilter.Meta.fields
```

Comment:
You can probably omit this row as keeps the same value.

Reply:
Good point, thanks for spotting this.